plugins:foreman_cockpit - Release 2.0.0 (RPM)

dLobatog opened the pull request.

domcleal commented on rubygem-foreman_cockpit/rubygem-foreman_cockpit.spec:
```diff
@@ -91,6 +91,9 @@ cp -a .%{gem_dir}/* \
 exit 0
 
 %changelog
+* Sun Apr 10 2016 Daniel Lobato Garcia <elobatocs@gmail.com> 2.0.0-1.fm1_10
```
Please leave this out, it won't be for fm1_10 and the changelog will be generated by tito.

dLobatog commented:
Updated - now it's 2.0.1 as I updated the list of families to fix a bug - https://github.com/theforeman/foreman_cockpit/commit/de0713fe67e6e02f452b5588ae9ab721af4b99dd

domcleal commented:
Thanks, merged and will build for 1.9 upwards, per the comment at #1083.
